envoy/templates/pdb.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
```

Comment:
PDB is also depcated in v1beta1, so it should be supported.
https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/2136